stetho/src/main/java/com/facebook/stetho/inspector/network/NetworkPeerManager.java

```diff
@@ -21,23 +21,31 @@
 
   private final ResponseBodyFileManager mResponseBodyFileManager;
 
+  private final AsyncPrettyPrinterRegistry mAsyncPrettyPrinterRegistry;
```

Comment:
I'm not sure why this is being placed into Network's PeerManager ... what will it be used for here?